app/mailers/school_mailer.rb

```diff
@@ -79,52 +79,35 @@ def nomination_confirmation_email(sit_profile:, school:, start_url:, step_by_ste
 
   # This email is sent to the school which has been reported by the lead provider and which hasn't yet nominated SIT
   # It also contains request to nominate SIT.
-  def school_partnership_notification_email(recipient:, partnership:, challenge_url:, nominate_url:)
-    template_mail(
+  def school_partnership_notification_email(partnership:, access_token:, reminder: false, campaign: nil)
+    email = template_mail(
       SCHOOL_PARTNERSHIP_NOTIFICATION_EMAIL_TEMPLATE,
-      to: recipient,
+      to: partnership.school.contact_email.presence || "ecf-tech@digital.education.gov.uk",
```

Comment:
What's the motivation for sending to us here?

Reply:
I'm not 100% sure, but I wanted to avoid as many changes in behaviour as possible. I think the reasoning here might be that we want to be notified if the email fails? Or we want to prevent delivery errors in the service? I'll try to find out :)